lab_2_tokenize_by_bpe/main.py

```diff
@@ -14,6 +14,23 @@ def prepare_word(
     :param end_of_word: a token that signifies the end of word
     :return: preprocessed word
     """
+    if not (isinstance(raw_word, str)
+            and isinstance(start_of_word, (str, type(None)))
+            and isinstance(end_of_word, str)
+    ):
+        return None
+
+    word_tokens = []
+
+    if start_of_word is not None:
+        word_tokens.append(start_of_word)
+
+    for char in raw_word:
```

Comment:
Can we extend the `word_tokens` with the characters of the word without writing a `for` loop?

Reply:
Still applies